BuildScripts/CopyProjDirectory.bat

```diff
@@ -0,0 +1,6 @@
+rem File "d:\checkouts\D-FAST\D-FAST_Morphological_Impact\dfastmi.dist\pyproj\datadir.py", line 109, in get_data_dir
```

Comment:
willen we dit er zo bijzetten?

miss handig om aan te geven dat nuitka deze files in zijn build nodig heeft en naar verwijst?

Reply:
Op z'n minst aan het begin van de batch file aangeven wat die comments betekenen. Telkens een runtime error message, en volgens de file copy die nodig is om de runtime error te voorkomen. De Nuitka build ziet wel dependencies naar de (meeste) libraries, maar niet als een library nog andere bestanden nodig heeft.